android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -28,7 +28,9 @@ import java.util.concurrent.TimeUnit
 class MusicService : HeadlessJsTaskService() {
     private lateinit var player: QueuedAudioPlayer
     private val handler = Handler(Looper.getMainLooper())
+
     var stopWithApp = false
+        private set
```

Comment:
I think it's good for this to only be settable inside the `MusicService`, as `updateOptions` is the function that is supposed to change this flag.